src/libraries/Microsoft.PowerFx.Core/Public/Config/VersionHash.cs

```diff
@@ -33,11 +34,15 @@ private VersionHash(int value)
             _value = value;
         }
 
+        private static int _hashStarter;
+
         public static VersionHash New()
         {
             // Give psuedo-random seed.
-            // This can help protect against 2 objects accidentally appearing the same. 
-            var value = (int)DateTime.UtcNow.Ticks;
```

Comment:
Remove use of the timer here in favor of something more deterministic. 

2 calls can reasonably happen at the same tick - new test below catches and fails.